kolibri/plugins/document_pdf_render/assets/src/views/index.vue

```diff
@@ -13,8 +13,12 @@
       <mat-svg v-if="isFullscreen" class="icon" category="navigation" name="fullscreen_exit"/>
       <mat-svg v-else class="icon" category="navigation" name="fullscreen"/>
     </icon-button>
+
+    <icon-button text="Zoom In" @click="zoomIn()"/>
+    <icon-button text="Zoom out" @click="zoomOut()"/>
+
     <div ref="pdfcontainer" class="pdfcontainer" @scroll="checkPages">
-      <progress-bar v-if="loading" class="progress-bar" :show-percentage="true" :progress="progress"/>
+      <progress-bar v-if="documentLoading" class="progress-bar" :show-percentage="true" :progress="progress"/>
```

Comment:
Just using a computed prop here. Thinking about going back to manually setting to have the page rendered on arrival